roles/redpanda_broker/templates/configs/defaults.j2

```diff
@@ -26,7 +30,8 @@
       "kafka_api": [
         {
           "address": "{{ hostvars[inventory_hostname].private_ip }}",
-          "port": "{{ redpanda_kafka_port }}"
+          "port": "{{ redpanda_kafka_port }}"{% if enable_sasl|default(false) %},
+          "authentication_method": "sasl"{% endif %}
         }
       ],
       "admin": [
```

Comment:
We can use sasl for the admin API as basic auth -- do we need to change any configuration to opt into admin api sasl? Would we want separate configuration?

Reply:
How does that work with intracluster communication on admin api? I wasn't sure so stayed back from it